pokebattle/players.py

```diff
@@ -0,0 +1,40 @@
+from nameko.rpc import rpc
+
+class PlayersService(object):
+    name = "players_service"
+
+    _players = []
+
+    @rpc
+    def new_player(self, name: str):
```

Comment:
Let's not use the types annotations for now, it will enable us to make it work on py2.7 which I'm using currently.
What do you think?